pkg/gops/cell.go

```diff
@@ -32,10 +32,13 @@ type GopsConfig struct {
 }
 
 func (def GopsConfig) Flags(flags *pflag.FlagSet) {
-	flags.Uint16(option.GopsPort, def.GopsPort, "Port for gops server to listen on")
+	flags.Uint16(option.GopsPort, def.GopsPort, "Port for gops server to listen on, set to 0 to disable")
```

Comment:
Port 0 often means letting the OS choose the port instead of specifying it explicitly. Can we instead have an new config flag to disable gops explicitly rather than relying on the port?

Reply:
sure